### PR TITLE
chore: add connection port and add auth into query-runner

### PR DESCRIPTION
**Pull request by DanielHe4rt**
In this Pull Request I brought some base implementation of using connection `ports` in a distinct field. 

Also I added the ability to accept connections with authentication:

```ts
// prepare the object for upcomming connections
let connectionObject = {
	nodes: [`${inputConnection.host}:${inputConnection.port}`],
} as ClusterConfig;

if (inputConnection.username && inputConnection.password) {
	connectionObject.auth = {
		username: inputConnection.username,
		password: inputConnection.password
	} as Auth;
}



// if a connection was never open and stored, then create a new connection
if (!connection) {
	const cluster = new Cluster(connectionObject);
	session = await cluster.connect();
	connections.push({
		...(inputConnection as Connection),
		session,
	});
	return session;
}
```

### CODE REVIEW

File: src/actions/execute-query.ts [hunk 1]
```diff
@@ -50,20 +60,31 @@ export const executeQueryAction = actionClient
  */
 export const getSession = async (inputConnection: Partial<Connection>) => {
 	const connection = connections.find((c) => c.id === inputConnection.id);
-
+	console.log(`${inputConnection.host}:${inputConnection.port}`);
```
Comment (Daniel-Boll) [inline src/actions/execute-query.ts:63]: Unnecessary log

File: src/actions/execute-query.ts [hunk 2]
```diff
@@ -50,20 +60,31 @@ export const executeQueryAction = actionClient
...
+	let connectionObject = {
+		nodes: [`${inputConnection.host}:${inputConnection.port}`],
+	} as ClusterConfig;
+
+	if (inputConnection.username && inputConnection.password) {
+		connectionObject.auth = {
+			username: inputConnection.username,
+			password: inputConnection.password
+		} as Auth;
+	}
```
Comment (Daniel-Boll) [inline src/actions/execute-query.ts:76]:
```suggestion
	const connectionObject: ClusterConfig = {
		nodes: [`${inputConnection.host}:${inputConnection.port}`],
		...((inputConnection.username && inputConnection.password) && {
			username: inputConnection.username,
			password: inputConnection.password
		} as Auth)
	};
```

File: src/app/(main)/connections/_components/connection-table.tsx [hunk 3]
```diff
@@ -19,8 +19,8 @@ import {
   TableHeader,
   TableRow,
 } from "@scylla-studio/components/ui/table";
-import type { Connection } from "@scylla-studio/lib/internal-db/connections";
-import { useEffect, useState, useTransition } from "react";
+import type {Connection} from "@scylla-studio/lib/internal-db/connections";
+import {useEffect, useState, useTransition} from "react";
```
Comment (Daniel-Boll) [inline src/app/(main)/connections/_components/connection-table.tsx:23]: Your editor be removing spaces [😭](https://emojipedia.org/loudly-crying-face)

File: src/app/(main)/connections/_components/modal.tsx [hunk 4]
```diff
@@ -58,9 +61,9 @@ interface NewConnectionModalProps {
 }
 
 export default function NewConnectionModal({
-  onSave,
-  connectionToEdit,
-}: NewConnectionModalProps) {
+                                             onSave,
+                                             connectionToEdit,
+                                           }: NewConnectionModalProps) {
```
Comment (Daniel-Boll) [inline src/app/(main)/connections/_components/modal.tsx:66]: Wut

**Review by Daniel-Boll — COMMENTED** [review]
We gotta setup the formatter for this project @Matozinho

**Review by Daniel-Boll — APPROVED** [review]

**Review by gvieira18 — APPROVED** [review]

File: src/app/(main)/connections/_components/connection-table.tsx (hunk 3 above)
Comment (DanielHe4rt) [reply src/app/(main)/connections/_components/connection-table.tsx]: i'm sorry man i'll tell him to not remove next time ok?

**Pull request merged**